packages/cosmos/src/client.rs

```diff
@@ -510,6 +519,20 @@ impl Cosmos {
         gas * self.pool.manager().coins_per_kgas / 1000
     }
 
+    /// Gas estimation is not perfect, and we need to adjust it by a multiplier to account for drift
+    /// Since we're already estimating and padding, the loss of precision from f64 to u64 is negligible
+    pub fn estimate_gas(&self, simulated_gas_used: u64) -> u64 {
```

Comment:
Maybe rename this from `estimate_gas`? It's not so much estimating as adding a buffer to an existing estimate.

Reply:
good point, went with `multiply_gas` in https://github.com/Levana-Protocol/levana-cosmos-rs/pull/8/commits/c9d2c1149eafcfad0d7ffea897f6fd12525b0979 to stick with the "multiplier" convention

Reply:
ended up getting rid of this function, it doesn't really make sense on its own, moved that calculation inline as part of simulating gas